src/Extensions/MemberExtension.php

```diff
@@ -55,7 +55,7 @@ public function updateCMSFields(FieldList $fields)
             'Root.Main',
             SessionManagerField::create(
                 'LoginSessions',
-                _t(__CLASS__ . '.DEVICES', 'Authenticated devices'),
+                _t(__CLASS__ . '.DEVICES', 'Login sessions'),
```

Comment:
The key should match the actual source string.

```suggestion
                _t(__CLASS__ . '.LOGIN_SESSIONS', 'Login sessions'),
```

Alternatively, you could do this and update the `singular_name` on Login Session to `Login session`.
```suggestion
                LoginSession::singleton()->plural_name(),
```

Reply:
> Run sake dev/tasks/i18nTextCollectorTask so all the translation strings are picked up.

That's done now.